crates/noirc_frontend/src/ast/expression.rs

```diff
@@ -643,7 +644,8 @@ impl FunctionDefinition {
             .collect();
         FunctionDefinition {
             name: name.clone(),
-            attribute: None,
+            // TODO(MD): is this the best way to do this? -> It was previously just None
+            attributes: Attributes::empty(),
```

Comment:
Yes I think so, since we now have a vector, it being empty is the same as having no attribute